tuning/scripts/stage_tuning.sh

```diff
@@ -1,86 +1,87 @@
 #!/bin/bash
 
-
-make_tensile_tuning () {
-
-    local FULLFILENAME=$1
-
-    local BUILDNAME="build"
-
-    local FILENAME=$(basename "$FULLFILENAME")
-    local FNAME="${FILENAME%.*}"
-    local FILEPATH=$(ls $SOURCE/$FILENAME)
-
-    local WORKINGPATHNAME=${BUILDNAME}-${FNAME}
-    local WORKINGPATH=${DESTINATION}/${WORKINGPATHNAME}
-
-    mkdir -p $WORKINGPATH
-    cp $FILEPATH $WORKINGPATH
-    pushd ${WORKINGPATH} > /dev/null
-    echo "#!/bin/sh" > doit.sh
-    echo "if [ ! -d 3_LibraryLogic ] || [ \$(ls -A 3_LibraryLogic | wc -c) -eq 0 ]; then" >> doit.sh
-    echo "  touch time.begin" >> doit.sh
-    echo "  ../Tensile/bin/Tensile $FILENAME ./ > make.out 2>&1" >> doit.sh
-    echo "  touch time.end" >> doit.sh
-    echo "fi" >> doit.sh
-
-    chmod +x doit.sh
-    popd > /dev/null
+function make_tensile_tuning() {
+
```

Comment:
May be out of scope for this review, but this is another example of a script that may be better suited to a Python environment. This generates a bash script that is only useful for that one specific logic file - IMO it makes the workflow pretty rigid.